pkg/kubelet/kubelet.go

```diff
@@ -319,6 +327,26 @@ type Dependencies struct {
 	useLegacyCadvisorStats bool
 }
 
+// MakeCrashLoopBackOffDecay configures the backoff maximum to be used
+// by kubelet for container restarts depending on the alpha gates
+// and kubelet configuration set
+func MakeCrashLoopBackOffDecay(kubeCfg *kubeletconfiginternal.KubeletConfiguration) (time.Duration, time.Duration) {
```

Comment:
maybe rename this to `newCrashLoopBackOff` and have it return `flowcontrol.NewBackOff(base, boMax)`?

Reply:
I changed the name but opted not to change the signature because it made the unittests more difficult since the backoff attributes are private